.changelog/unreleased/imrpovements/3442-limit-pgf-stewards.md

```diff
@@ -0,0 +1,2 @@
+- Enforce an upper limit on the number of PGF stewards allowed to exist at a
```

Comment:
Small typo in the name of this file `imrpovements`

Reply:
fixed thx